app/models/zine.rb

```diff
@@ -71,11 +71,11 @@ def generate_pdf_from_legacy_id
         "http://assets.zinedistro.org/zines/pdfs/#{legacy_id}.pdf"
   end
 
-  def incriment_author_cache_counter(record)
+  def increment_author_cache_counter(record)
     update_zine_counter(1)
   end
 
-  def decriment_author_cache_counter(record)
+  def decrement_author_cache_counter(record)
```

Comment:
Unused method argument - `record`. If it's necessary, use `_` or `_record` as an argument name to indicate that it won't be used. You can also write as `decrement_author_cache_counter(*)` if you want the method to accept any arguments but don't care about them.